RecoMuon/MuonIdentification/plugins/BuildFile.xml

```diff
@@ -12,6 +12,7 @@
   <use   name="Geometry/Records"/>
   <use   name="Geometry/CaloTopology"/>
   <use   name="PhysicsTools/IsolationAlgos"/>
+  <use   name="PhysicsTools/SelectorUtils"/>
```

Comment:
@slava77 - so not specific to this PR - but we should move things in PhysicsTools that Reco* needs into CommonTools. That will make it much easier to disintangle dependencies